src/containers/StudyDetail/MbtiRecommand/MbtiRecommend.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  MbtiRecommandContainer,
+  MbtiTagWrapper,
+} from '@src/containers/StudyDetail/MbtiRecommand/style';
+import { Typography } from '@mui/material';
+import { MbtiTag } from '@components';
+
+interface Props {
+  preferredMbtis: string[];
+}
+
+function MbtiRecommend({ preferredMbtis }: Props) {
+  return (
+    <MbtiRecommandContainer>
+      <Typography variant='h5'>추천 MBTI</Typography>
```

Comment:
```suggestion
      <Typography variant='h5'>선호 MBTI</Typography>
```
스터디 생성자가 지정한 선호 MBTI를 보여주는 것이라 선호 MBTI가 맞는것 같습니다!